audio_capturer:
  ros__parameters:
    MIC_DEVICE_NAME: "default"
    MIC_INPUT_CHANNELS: 32
    MIC_OUT_CHANNELS: 32
    USE_RESPEAKER: False
    publish_topic: "/rawAudioChunk"
